src/app/modules/item/pages/item-edit-advanced-parameters/item-edit-advanced-parameters.component.ts

```diff
@@ -77,7 +76,7 @@ export class ItemEditAdvancedParametersComponent implements OnInit {
     const enteringTimeMin = this.parentForm?.get('entering_time_min')?.value as string;
     const enteringTimeMinDate = new Date(enteringTimeMin);
     this.parentForm?.get('entering_time_max')?.patchValue(
-      new Date(enteringTimeMinDate.getTime() + INIT_MAX_RANGE_MS)
+      new Date(enteringTimeMinDate.getTime() + (HOURS * 3600 * 1000))
```

Comment:
actually my remark was more to use `HOURS` from `duration.ts` (not to introduce again this 3600*1000)